Source/WebCore/platform/graphics/coretext/FontPlatformDataCoreText.cpp

```diff
@@ -361,15 +361,15 @@ std::optional<FontPlatformSerializedAttributes> FontPlatformSerializedAttributes
         return WTFMove(vector);
     };
 
-    auto paletteColors = static_cast<CFDictionaryRef>(CFDictionaryGetValue(dictionary, kCTFontPaletteColorsAttribute));
-    result.paletteColors = pairExtractor.template operator()<CGColorRef>(paletteColors, CGColorGetTypeID());
+    RetainPtr paletteColors = static_cast<CFDictionaryRef>(CFDictionaryGetValue(dictionary, kCTFontPaletteColorsAttribute));
```

Comment:
These should use a checked CF cast of some sort rather than `static_cast`. They have handwritten type checking code but that's what checked and dynamic CF casts are now for.